doc/governance/org-docs/CHARTER.md

```diff
@@ -0,0 +1,73 @@
+# Charter for the HoloViz Organization
+
+This is the organizational charter for the HoloViz Organization. The HoloViz Organization is an open-source and collaborative effort with a mission to develop solutions for a wide range of visualization problems in Python.
+
+## 1. Terminology
+- "HoloViz" or "HoloViz Community" or "Community": The entire HoloViz ecosystem of Users, Contributors, and Projects
+- "HoloViz Organization" or "Organization": The voting members of HoloViz Committees and Projects
+- "HoloViz GitHub Organization": The Github organization at https://github.com/holoviz
+- "HoloViz" repo: The specific repository at https://github.com/holoviz/holoviz
+- "HoloViz.org": The website at https://holoviz.org/
```

Comment:
Should we add our other communication channels somewhere like Discourse, Gitter, LinkedIn, and Twitter? I'm not sure if these should be included somewhere or not.

Reply:
yes, I think I will add it to the definition of HoloViz Community to include all those channels in the gov scope

Reply:
I'll also add a definition for "'HoloViz Project' or 'Organization Project' or 'Project'"